### ISSUE TITLE
Testing and linting for both tech, ux and accessibility

### ISSUE THREAD

**Issue by Camulos** [digdir/designsystem#4612]
### Description

Should designsystemet offer some ready to use tools for testing and marking of technical failty implementation, ux-issues that is visible in code and accessibility issues that is visible in code?
Example shaming-css: https://heydonworks.com/article/testing-html-with-modern-css/
